diff --git a/services/core/java/com/android/server/pm/ComputerEngine.java b/services/core/java/com/android/server/pm/ComputerEngine.java
index 26deb6ccdc09..414e826476a5 100644
--- a/services/core/java/com/android/server/pm/ComputerEngine.java
+++ b/services/core/java/com/android/server/pm/ComputerEngine.java
@@ -1486,9 +1486,10 @@ public class ComputerEngine implements Computer {
     private static native boolean isDebuggable();
 
     public static boolean isMicrogSigned(AndroidPackage p) {
+        /*
         if (!isDebuggable()) {
             return false;
-        }
+        }*/
 
         // Allowlist the following apps:
         // * com.android.vending - microG Companion
